pyproject.toml

```diff
@@ -91,12 +92,15 @@ dev = [
     "typeguard==2.13.3",
     "ruff==0.1.13",
     "sshconf==0.2.5",
-    "pycolmap>=0.3.0",  # NOTE: pycolmap==0.3.0 is not available on newer python versions
+    # TODO(1480) enable when pycolmap windows wheels are available
+    # "pycolmap>=0.3.0",  # NOTE: pycolmap==0.3.0 is not available on newer python versions
```

Comment:
Let's put this in a different PR, as it's not so related to adding support for EyefulTower.

Reply:
Sure, I can pull it into a separate PR. I put it here since I was doing my development on Windows and this was blocking me from installing the environment, but now that it's done I can easily make a separate PR. Coming soon.